### DISCUSSION TITLE
🧵 Feature request: Nest @GitBook SlackBot replies in Slack thread

### DISCUSSION THREAD

**Discussion by jkroll-deepgram** [GitbookIO/community#481] [Feature Requests]
- **Current state**: When a user @'s GitBook in Slack via `@GitBook <question>`, GitBook replies in a subsequent message. Its response takes up a large amount of screen real estate, and is effectively a page-long reply. Any individual may ask a question in a Slack channel, and no one else may be interested in the reply, yet it clutters the channel.

- **Requested feature**: When a user posts in Slack with `@GitBook`, have GitBook nest its reply in-thread, to keep the discussion compact and streamlined. This is a best practice for humans using Slack, and it should apply to Slackbots too. 
    - Also as a nice-to-have, it would be a bonus to make its replies use as few lines as possible, instead of padding with whitespace, suggestions, and the original query repeated multiple times.

**Comment by Flochat** [comment]
Hey this may help: So I assume you're using user query to get answers back to users via LensAI. If so, what we have found is that you can prefix the query with a basic prompt: eg, respond in a supportive tone. This will provide a better response. On top of that as you mention that the replies are very long and take up a large space, again you can prefix the user query, with a simple prompt: Reply in one sentence {{userquery}}. Both of these type of prefix to the query will also reduce the same answer coming back.. The main reason we see to why it does reply the same answer again, is simply down to prefixing the query prompt.. without the prompt we found the answer to either be very bland, longer context or not fetch data at all. 

Hope that helps... Pete
